Manual/ErrorExplanations/README.md

```diff
@@ -0,0 +1,124 @@
+# Writing Error Explanations
+
+Error explanations give more context and discussion than is reasonable to
+include in an actual error message. The error message in the Lean InfoView
+contains a link to the error explanation for further reading.
+
+An error explanation is just a manual page obeying a few conventions:
```

Comment:
Do we validate this anywhere?

If not, I can help write a quick check for these conventions that runs in CI after the holidays.